x-pack/test/search_sessions_integration/tests/apps/discover/async_search.ts

```diff
@@ -131,6 +131,19 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
 
       expect(searchesCountBeforeRestore).to.be(searchesCountAfterRestore); // no new searches started during restore
     });
+
```

Comment:
ℹ️ I had removed it in a previous PR (when we were using the same histogram from the dataview mode). Now that the histogram is constructed from an ES|QL query, I am adding this back